tests/test_hypothesis_plugin.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from hypothesis import given, settings, strategies as st
+
+import pydantic
+from pydantic.networks import import_email_validator
+
+
+class MiscModel(pydantic.BaseModel):
```

Comment:
best if possible for models to be declared within tests or fixtures (I know we don't currently stick to this 100%, but it would be good if we could in new code).

You can do something like

```py
def gen_models():
    class MiscModel(pydantic.BaseModel):
        ...
    yield MiscModel

    class StringsModel(pydantic.BaseModel):
        ...
    yield StringsModel
    ...


...

@pytest.mark.parametrize('model', gen_models())
```

That way the models should only be created when the test(s) are executed.